docs/concepts/error-pages.md

```diff
@@ -22,6 +24,39 @@ export default function NotFoundPage({ url }: UnknownPageProps) {
 }
 ```
 
+#### Manually render 404
+
+The `_404.tsx` file will be invoke when not routes matches the URL. In some
+cases, you would want to manually trigger the rendering of the 404 page with
+`renderNotFound`.
```

Comment:
```suggestion
The `_404.tsx` file will be invoked automatically when no route matches the URL. In some
cases, one needs to manually trigger the rendering of the 404 page, for example when the route did match, but the requested resource does not exist. This can be achieved with
`ctx.renderNotFound`.
```